main.go

```diff
@@ -52,8 +52,8 @@ func main() {
 	// Example route that takes one rest style option
 	router.GET("/hello/:name", hello)
 
-	// Serve static assets via the "www" directory
-	router.NotFound = http.FileServer(assetFS())
+	// Serve static assets via the "static" directory
+	router.ServeFiles("/static/*filepath", assetFS())
```

Comment:
No, it works as intended. Both static content and the test route were invoked/tested successfully.